packages/charm.rb

```diff
@@ -9,25 +9,15 @@ class Charm < Package
   source_url 'http://downloads.sourceforge.net/project/ljcharm/charm/charm-1.9.2/charm-1.9.2.tar.gz'
   source_sha256 'ae2d536303e770ceb002f69c1e71f6a8532c8d5dd075d2f5c1bf09ad02c3c015'
 
```

Comment:
Bump version?

Reply:
Done, waiting for futher reviews to push.